prov/efa/src/efa_user_info.c

```diff
@@ -354,6 +356,36 @@ int efa_user_info_alter_rxr(struct fi_info *info, const struct fi_info *hints)
 
 			info->domain_attr->mr_mode |= FI_MR_HMEM;
 
+			support_atomic = false;
+			if (hmem_ops[FI_HMEM_CUDA].initialized == true) {
+				/*
+				  We are making an assumption here that the remote we are performing atomic
+				  operations on is the same as us. This might not be true for atomics over
+				  client/server model using EFA.  We do not currently have any users trying
+				  to do this, so we are not worried about this assumption yet.
+				*/
+				fi_param_get_bool(NULL, "hmem_cuda_use_gdrcopy", &hmem_cuda_use_gdrcopy);
+				fi_param_get_bool(NULL, "hmem_cuda_enable_xfer", &hmem_cuda_enable_xfer);
```

Comment:
should use the function `cuda_get_xfer_setting()`

Reply:
fixed.